src/python_ms_core/core/storage/providers/azure/azure_file_entity.py

```diff
@@ -21,6 +21,7 @@ def get_body_text(self):
 
     @ExceptionHandler.decorated
     def upload(self, upload_stream):
+        # Infer the content type here or at the place where upload is happening
         upload_file = self.blob_client.upload_blob(self.file_path, upload_stream)
```

Comment:
@susrisha can you add `overwrite=True` in upload_blob function.

This will not fail if the same file is uploaded